en/dev_setup/dev_env_mac.md

```diff
@@ -85,8 +85,11 @@ brew install --cask adoptopenjdk14
 brew install px4-sim-jmavsim
 ```
 
-:::note
-jMAVSim for PX4 v1.11 and earlier required Java 8.
+:::warning
+jMAVSim for PX4 v1.11 and beyond we require at least JDK 15.
```

Comment:
@mrpollo The problem, as discussed in related issue you closed, is that this may not be true. 
JDK 14 should work and there is some other problem here. My concern is that this will work for now, but might break for the same reason magically at some other point - just as JDK 14 has ... it used to work.

If no one is going to debug this properly (which it looks like) then yes, we can update. But you will need to update the previous lines from 76 as well to specify JDK15

I have subedit this section

Reply:
Actually I have updated the section JDK 14 above as well. I'll add my "concern" as a public comment and merge this.